concrete/blocks/express_entry_list/controller.php

```diff
@@ -127,21 +137,39 @@ public function edit()
                 $this->set('linkedPropertiesSelected', $linkedPropertiesSelected);
                 $this->set('searchPropertiesSelected', $searchPropertiesSelected);
                 $this->set('searchProperties', $searchProperties);
+                $this->set('searchAssociationsSelected', $searchAssociationsSelected);
+                $this->set('searchAssociations', $searchAssociations);
             }
         }
-
     }
 
     protected function getSearchPropertiesJsonArray($entity)
     {
         $attributes = $entity->getAttributeKeyCategory()->getList();
-        $select = array();
-        foreach($attributes as $ak) {
-            $o = new \stdClass;
+        $select = [];
+        foreach ($attributes as $ak) {
+            $o = new \stdClass();
             $o->akID = $ak->getAttributeKeyID();
             $o->akName = $ak->getAttributeKeyDisplayName();
             $select[] = $o;
         }
+
+        return $select;
+    }
+
+    protected function getSearchAssociationsJsonArray($entity)
+    {
+        $associations = $entity->getAssociations();
+        $select = [];
+        foreach ($associations as $association) {
+            if ($association instanceof ManyToManyAssociation || $association instanceof ManyToOneAssociation) {
+                $o = new \stdClass();
+                $o->associationID = $association->getId();
+                $o->associationName = $association->getTargetEntity()->getName();
```

Comment:
I wonder how this eventually plays out if you use `<script>alert('test')</script>` as a name in the associated entity.

Reply:
I can add `getDisplayName` same as attribute key. Can I do it?

Reply:
Oops, `getEntityDisplayName` already existed